src/test/java/jimmy/mcgymmy/testutil/TypicalFoods.java

```diff
@@ -13,44 +14,31 @@
-            .withProtein("253").withFat("123")
-            .withCarb("456").withDate("20 Apr 2020")
-            .withTags("lunch").build();
-    public static final Food NASI_LEMAK = new FoodBuilder().withName(new Name("Nasi Alamak"))
-            .withProtein("432").withFat("321")
-            .withCarb("123").withDate("2 Sep 2020")
-            .withTags("dinner", "lunch").build();
-    public static final Food CRISPY_FRIED_FISH = new FoodBuilder().withName(new Name("Crispy Fried Fish"))
-            .withProtein("563").withFat("456").withCarb("654")
-            .withDate("13 Oct 2020").build();
-    public static final Food DANISH_COOKIES = new FoodBuilder().withName(new Name("Danish Cookies"))
-            .withProtein("533").withFat("654")
-            .withCarb("456").withDate("1 Jan 2020")
-            .withTags("lunch").build();
-    public static final Food EGGS = new FoodBuilder().withName(new Name("Eggs"))
-            .withProtein("224").withFat("246").withCarb("810")
-            .withDate("7 Mar 2019").build();
-    public static final Food FRUIT_CAKE = new FoodBuilder().withName(new Name("Fruit Cake"))
-            .withProtein("427").withFat("987").withCarb("789")
-            .withDate("3 Feb 1998").build();
-    public static final Food GINGERBREAD = new FoodBuilder().withName(new Name("Gingerbread"))
-            .withProtein("442").withFat("789").withCarb("987")
-            .withDate("5 May 1973").build();
+    public static final String KEYWORD_MATCHING_RICE = "Rice"; // A keyword that matches RICE
+
+    private static Food chickenRice;
+    private static Food nasiLemak;
+    private static Food crispyFriedFish;
+    private static Food danishCookies;
+    private static Food eggs;
+    private static Food fruitCake;
+    private static Food gingerbread;
 
     // Manually added
-    public static final Food HOT_PLATE = new FoodBuilder().withName(new Name("Hot Plate")).withProtein("424")
-            .withFat("234").withCarb("234").build();
-    public static final Food INDOMEE = new FoodBuilder().withName(new Name("Indomee")).withProtein("131")
-            .withFat("234").withCarb("234").build();
+    private static Food hotPlate;
+    private static Food indomee;
 
     // Manually added - Food's details found in {@code CommandTestUtil}
-    public static final Food APPLE = new FoodBuilder().withName(new Name("Apple")).withProtein("888")
-            .withFat("234").withCarb("234").build();
-    public static final Food BEANS = new FoodBuilder().withName(new Name("beans")).withProtein("888")
-            .withFat("234").withCarb("234").build();
+    private static Food apple;
+    private static Food beans;
 
-    public static final String KEYWORD_MATCHING_RICE = "Rice"; // A keyword that matches RICE
+    static {
+        try {
+            generateTypicalFoods();
+        } catch (IllegalValueException e) {
+            assert false : "Error in Typical foods";
```

Comment:
Could have placed this in `FoodBuilder` where you made a new function `BuildUnsafe` which throws a `RuntimeException` or `IllegalValueException` just for testing, to reduce the code changed. but this works too.

Too much effort to change to the above suggestion, so i guess we can leave this.